runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -1465,13 +1465,14 @@ jvmtiGetOSThreadID(jvmtiEnv* jvmti_env, ...)
 		ENSURE_PHASE_START_OR_LIVE(jvmti_env);
 		ENSURE_NON_NULL(threadid_ptr);
 
+		rc = getVMThread(
+				currentThread, thread, &targetThread,
 #if JAVA_SPEC_VERSION >= 19
```

Comment:
This ifdef is unnecessary as the vthread errors only happen on 19+ anyway.

Reply:
JVMTI spec has introduced `JVMTI_ERROR_UNSUPPORTED_OPERATION` in JDK19+.